src/app/games/games.service.ts

```diff
@@ -24,4 +24,8 @@ export class GamesService implements IGamesService {
   getGameCSV(filename: string): Observable<string> {
     return this.http.get(`/assets/csv/${filename}.csv`, { responseType: "text" });
   }
+
+  getGameFAQ(filename: string): Observable<string> {
```

Comment:
Please put a definition in the interface for this method, with appropriate documentation, to be consistent with the other methods.

Reply:
What's that for?

Reply:
If we ever want to mock this service for testing purposes, we will want to have it implement the same interface that this service is currently using. Hence why they have definitions in an interface. If nothing else, as aforementioned, it's consistent with the other methods.